### DISCUSSION TITLE
Exclude an Element from Prose Style Applying

### DISCUSSION THREAD

**Discussion by ahangarha** [tailwindlabs/tailwindcss#4095] [Ideas]
I find myself sometimes in a situation that I need to manipulate style of one particular element (let's say a link) but cannot bucause prose get over all other classes applied to the element.

I thought maybe it would be good to have `no-prose` class which can exceptionally protect the element from prose styling. I think that can be done by adding `<SELECTOR>:not(.no-prose)` to each selector in prose styling. So that if one use this class on an element, prose wont touch it.

**Update:** I forgot that I am sharing this on tailwindcss not typography plugin. So I need to mention clearly that I am talking about some tweak in tailwindcss/typography plugin.

**Comment by MartinHughes-BPC** [comment]
Hi @ahangarha 

If you can use the new 'jit' feature of Tailwind (I thoroughly recommend doing so - it's awesome 👍) you could use the important (!) feature to override the defaults. For example:

```
<article class="prose lg:prose-xl">
            <h1 class="!text-xs">Garlic bread with cheese: What the science tells us</h1>
            <p>
                For years parents have espoused the health benefits of eating garlic bread with cheese to their
                children, with the food earning such an iconic status in our culture that kids will often dress
                up as warm, cheesy loaf for Halloween.
            </p>
            <p>
                But a recent study shows that the celebrated appetizer may be linked to a series of rabies cases
                springing up around the country.
            </p>
            <!-- ... -->
</article>
```

Notice I've done !text-xs in the h1? This overrides the default from tailwind typography and applies text-xs to it!

Cheers,
Martin.

**Reply by ahangarha** [reply]
It would work. But I think still the typography plugin needs such feature to exclude some element from getting affected.

**Comment by bagsa** [comment]
[]()

**Reply by bagsa** [reply]
ًتن

**Comment by bagsa** [comment]
- [ ] ا[بب]()
